changelog.d/834.bugfux

```diff
@@ -0,0 +1 @@
+Include the domain name in the regular expression
```

Comment:
🫢 ITYM bugf*I*x

Reply:
Lol funniest typo I've made in a while. Try now